internal/osutils/shortcut/shortcut_windows.go

```diff
@@ -17,12 +17,13 @@ type Shortcut struct {
 	dir      string
 	name     string
 	target   string
+	args     string
 	dispatch *ole.IDispatch
 }
 
-func New(dir, name, target string) *Shortcut {
+func New(dir, name, target, args string) *Shortcut {
```

Comment:
functions with an `args` argument are typically variadic. That will make the last argument optional and you won't have to update existing calls of this function.

Reply:
I thought about this, but I didn't want args to be more than one in length (so truly an optional argument, not variadic). Nevertheless, I've implemented your suggestion.